CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+### Security fixes
+
+* Fixed SHA-1 collision attacks not being detected.
+  ([GHSA-794x-2rpg-rfgr](https://github.com/jj-vcs/jj/security/advisories/GHSA-794x-2rpg-rfgr))
```

Comment:
Wrong link (or the advisory is not public yet)?

Reply:
Yes, not public yet. (But it’s just the “aimed at Jujutsu users/distributors” version of https://github.com/GitoxideLabs/gitoxide/security/advisories/GHSA-2frx-2596-x5r6.)

Reply:
I think @martinvonz can [give the maintainers team permission to manage advisories](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/managing-security-managers-in-your-organization), BTW. Probably a good idea.

Reply:
I think I've done that now